src/snowflake/cli/_plugins/nativeapp/codegen/compiler.py

```diff
@@ -155,5 +155,5 @@ def _should_invoke_processors(self):
 
     def _is_enabled(self, processor: ProcessorMapping) -> bool:
         if processor.name.lower() == NA_SETUP_PROCESSOR:
-            return FeatureFlag.ENABLE_NATIVE_APP_PYTHON_SETUP.is_enabled()
+            return FeatureFlag.ENABLE_NATIVE_APP_PYTHON_SETUP.get_flag_value() is True
```

Comment:
How do you suggest doing this?
- is_enabled: True if set to true, False if set to False, None otherwise?
- is_disabled: True if set to false, False if set to True, None otherwise?

OR:
- is_enabled: True if set to true, False otherwise (including None).
- is_disabled: False if set to false, False otherwise (including None).
Optional: - is_set: True if explicitly set.

WDYT?